hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/config/ConfigServer.java

```diff
@@ -124,6 +120,11 @@ public synchronized void handle(
       model.put("dataDir", hdfsFrameworkConfig.getDataDir());
       model.put("haZookeeperQuorum", hdfsFrameworkConfig.getHaZookeeperQuorum());
 
+      String nnNum = request.getParameter("nn");
```

Comment:
"nn" is used in a couple of places. Let's create a constant in HDFSConstants. It could be called NAMENODE_NUM_PARAM